testing/cucumber/features/linters/scenario_with_bad_name.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Scenario with bad name linter
+
+  As a reader of documentation
+  I want every scenario to have Given and When steps rather than Test, Verify or Check
+  So that the naming is consistent and intent clear
+
+
+  Scenario: Linting
+
+  Note: Also works on outlines.
+
+    Given a linter for tests with bad names
+    And the following feature:
+      """
+      Feature:
+
+        Scenario:
+          This scenario has a bad name
+      """
+    When it is linted
+    Then an error is reported:
+      | linter                    | problem                                                                                      | location         |
+      | ScenarioWithBadNameLinter | Prefer name your scenarios using "Given" and "When" rather than "test", "verify" or "check". | <path_to_file>:3 |
```

Comment:
Again, I'm not sure why the original message was referring to step keywords. How about something like `Avoid test-like words such as "test", "verify", or "check" in scenario names`?